### ISSUE TITLE
[Proposal] Create wiki pages for documentation

### ISSUE THREAD

**Issue by alan-agius4** [dherges/ng-packagr#733]
## Type of Issue

```
[ ] Bug Report
[x] Feature Request
```

## Description
With every release more features are added, and its kinda hard to keep track how to do something from the readme and the other md docs as they are growing longer and longer.

I believe that a wiki space should be created with organized pages.

Angular CLI Team have some good wiki pages/stores https://github.com/angular/angular-cli/wiki

**Comment by georgiee** [comment]
Yes I totally support this as I keep seeing me digging through the ng-package sources 👍 
I would like to document the whole API surface but also some good examples - as they are already present in the integration folder in the sources.

Instead of just demanding, such a piece of documentation is a perfect thing we could easily compile and provide @dherges as a draft to continue working on. Should we start outlining here what to would be nice to see in such a documentation?

I see something like this:
+ Configuration/API
+ Common Problems (Typescript Version, Barrel Files)
+ Best Practice (for example: manually copy over readme files)
+ Examples (Integration files as a base)
+ Cheat sheet (some compact tables)

**Comment by alan-agius4** [comment]
Wow nice list 😁. That should be a great start haha

**Comment by georgiee** [comment]
Good point, I have seen this too with tsconfig and totally understand why it's happening.

**Comment by dherges** [comment]
Hi @georgiee @alan-agius4 sound sperfectly reasonable. I don't know yet how this project will continue, whether it is standalone or in a github org or as part of the angular devkit. 

If you like to add documentation, wdyt about adding markdown files to the `docs` folder of this repo?

**Comment by alan-agius4** [comment]
@dherges I am fine with that as well. I'll try to do some documentation when I have some time.

**Comment by georgiee** [comment]
Yes, a docs folder is the perfect place compared to the github wiki and the like. I also try to collect some useful notes for the documentation. We could work on a draft in a shared google docs document instead of discussion it in Pull Requests. Is this feasible?
